src/payment/__snapshots__/PaymentPage.test.jsx.snap

```diff
@@ -8727,10 +8727,7 @@ exports[`<PaymentPage /> Renders correctly in various states should render the b
               </span>
               <span>
                 <span>
-                  <span>
```

Comment:
So that this changed is concerning - doesn't it mean that it used to have a currency cookie for MX and now it doesn't?

Reply:
This snapshot says "should render the basket in a different currency"

Reply:
Good catch. looking...

Reply:
Fixed